ui/appui-react/src/test/TestUtils.ts

```diff
@@ -30,7 +28,9 @@ const initialState: SampleAppState = {
   placeHolder: false,
 };
 
-/** @internal */
+/**
+ *
+ */
 export interface RootState {
```

Comment:
This seems like an unintentional change

Reply:
Ah good catch, auto-formatter added that back. I'll revert

Reply:
fixed

Reply:
`RootState` interface is now neither `@public`, nor `@internal`, but I'm not sure this matters

Reply:
It's a test utility so didn't think it mattered too much.